common/djangoapps/contentserver/test/test_contentserver.py

```diff
@@ -56,11 +62,13 @@ def setUpClass(cls):
         # A locked asset
         cls.locked_asset = cls.course_key.make_asset_key('asset', 'sample_static.txt')
         cls.url_locked = unicode(cls.locked_asset)
+        cls.url_locked_versioned = get_versioned_asset_url(unicode(cls.locked_asset))
```

Comment:
Well, following up on my first comment: we could totally version locked assets and use the same long TTL for them.  They would simply never be served from the CDN.

I'm going to change this so all asset URLs are versioned.